src/sql/plugins/publicdashboard/core/init_core_publicdashboard.sql

```diff
@@ -0,0 +1,15 @@
+
+--
+-- Data for table core_admin_right
+--
+DELETE FROM core_admin_right WHERE id_right = 'COMPONENTDASHBOARD_MANAGEMENT';
```

Comment:
All references to "COMPONENTDASHBOARD_..."  maybe replaced by "PUBLICDASHBOARD_..."